public/app/features/explore/Table/TableContainer.tsx

```diff
@@ -58,7 +58,11 @@ export class TableContainer extends PureComponent<Props> {
       name = data.refId || `${i}`;
     }
 
-    return name ? t('explore.table.title-with-name', 'Table - {{name}}', { name }) : t('explore.table.title', 'Table');
+    return name ? (
+        <Trans i18nKey="explore.table.title-with-name">
+          Table - {{name}}
+        </Trans>
+      ) : t('explore.table.title', 'Table');
```

Comment:
An alternative solution would overriding `escapeValue` in this specific case, like so:

```suggestion
    return name ?
      t('explore.table.title-with-name', 'Table - {{name}}', { name, interpolation: { escapeValue: false } }) :
      t('explore.table.title', 'Table');
```